src/Microsoft.AspNetCore.SignalR/SignalROptions.cs

```diff
@@ -1,20 +1,10 @@
 // Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Are we expecting any options here anytime soon? Or should we just remove this class for now?